pkg/subctl/cmd/join.go

```diff
@@ -166,12 +169,14 @@ func joinSubmarinerCluster(config *rest.Config, subctlData *datafile.SubctlData)
 		}
 	}
 
+	err := checkRequirements(config)
+	exitOnError("The cluster doesn't meet our requirements", err)
```

Comment:
Minor thing but the message here is redundant with the unsupported version message  and wouldn't apply to the other API failure messages. I suggest using `exitWithErrorMsg` and maybe clarifying the API failure messages.

Reply:
Done, thanks!